programmes/models.py

```diff
@@ -99,9 +99,23 @@ def map_string(cls, value):
         return value
 
 
-class OppenedNatureLogement(models.TextChoices):
+class OpenedNatureLogement(models.TextChoices):
```

Comment:
By "opened" you mean "extended" ? Ou peut-être qu'il s'agit des natures de logement "actifs" ?

En comparant, j'ai vu que le type `NatureLogement` avait juste `RHVS` en plus (et des labels différents) on pourrait peut-être fusionner, non ?

Reply:
Oui, c'est plutôt actif...